.github/workflows/release.yml

```diff
@@ -7,6 +7,7 @@ jobs:
   luarocks-upload:
     runs-on: ubuntu-20.04
     steps:
+      - uses: actions/checkout@v2
```

Comment:
lets use version `@v3`

i also just updated the rest of the CI

Reply:
Yup, just saw your comment in #435 :)